pkg/openconnect

```diff
@@ -0,0 +1,31 @@
+[mirrors]
+ftp://ftp.infradead.org/pub/openconnect/openconnect-8.04.tar.gz
+
+[vars]
+filesize=1914479
+sha512=3d1f335c5ac62cdcf874b0371e9ed939e5e44060d422b35120d0a6bb87f1a7cc4ffc783e6c65d11a9d5ef974c99e56107da837ee61a03f70d9397e077185050a
+pkgver=1
+desc="Open client for Cisco AnyConnect VPN, and more."
+
+[deps]
+vpnc-scripts-tarball
+
+[build]
+#patch -p1 < "$K"/some.patch
+#cp -f "$K"/config.sub .
+[ -n "$CROSS_COMPILE" ] && \
+  xconfflags="--host=$($CC -dumpmachine) \
+  --with-sysroot=$butch_root_dir"
+
+CPPFLAGS="-D_GNU_SOURCE" CFLAGS="$optcflags" CXXFLAGS="$optcflags" \
+LDFLAGS="$optldflags -Wl,-rpath-link=$butch_root_dir$butch_prefix/lib" \
+  ./configure -C --prefix="$butch_prefix" \
+		--disable-nls $xconfflags \
+		--sbindir="$butch_prefix"/bin
+
+#[ -n "$CROSS_COMPILE" ] && "$K"/libtool.cross.gen --sysroot="$butch_root_dir"
+
+make V=1 -j$MAKE_THREADS
+make DESTDIR="$butch_install_dir" install
+mkdir -p /etc/vpnc
+install -m644 "${K}/vpnc-script" /etc/vpnc/
```

Comment:
1) don't use /etc directly (grep other packages for `$dest`)
2) `install -Dm 644` creates all path components so an explicit mkdir isnt needed